apps/server/src/infra/rabbitmq/rabbitmq.config.ts

```diff
@@ -5,7 +5,9 @@ export const FilesPreviewExchange = Configuration.get('FILES_STORAGE__EXCHANGE')
 export const FilesStorageExchange = Configuration.get('FILES_STORAGE__EXCHANGE') as string;
 export const MailSendExchange = Configuration.get('MAIL_SEND_EXCHANGE') as string;
 export const AntivirusExchange = Configuration.get('ANTIVIRUS_EXCHANGE') as string;
+export const SchulconnexProvisioningExchange = Configuration.get('PROVISIONING_SCHULCONNEX_EXCHANGE') as string;
 export const RabbitMqURI = Configuration.get('RABBITMQ_URI') as string;
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface RabbitMqConfig {}
+export interface RabbitMqConfig {
+	RABBITMQ_URI: string;
+}
```

Comment:
Why this change?

Reply:
Because the RabbitMqModule needs the RABBITMQ_URI in order to work, so the interface should contain the variable. Otherwise it would not make sense to have this interface, since it is an empty interface that does not represent the required configuration of the module.